File: verification/verify_caballo2.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();

  await page.goto('http://localhost:8000/atermis-caballo2/index.html');

  // Wait a bit to ensure it loads completely
  await page.waitForTimeout(1000);

  // Capture the initial state
  await page.screenshot({ path: 'verification/atermis-caballo2-level1.png' });

  // Change level to 8x8
  await page.selectOption('#level-select', '8');
  await page.waitForTimeout(500);
  await page.screenshot({ path: 'verification/atermis-caballo2-level3.png' });

  await browser.close();
})();

File: verification/verify_caballo2.py
from playwright.sync_api import sync_playwright
import time

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto('http://localhost:8000/atermis-caballo2/index.html')
    time.sleep(1)
    page.screenshot(path='verification/atermis-caballo2-level1.png')

    page.select_option('#level-select', '8')
    time.sleep(0.5)
    page.screenshot(path='verification/atermis-caballo2-level3.png')

    browser.close()
